tests/unit/test_score_matching.py

```diff
@@ -53,33 +54,6 @@ class TestKernelDensityMatching(unittest.TestCase):
     Tests related to the class in ``score_matching.py``.
     """
 
```

Comment:
Why has test_tree_flatten been removed?

Reply:
Well spotted, I was going to convert the score_matching to equinox and must have forgotten to revert this change.